import-areas/import-areas.go

```diff
@@ -258,11 +285,25 @@ func postAreaQuery(query string) ([]models.Area, error) {
 		v.Set("query", query)
 
 		client := &http.Client{}
-		req, _ := http.NewRequest("POST", "http://statistics.data.gov.uk/sparql", strings.NewReader(v.Encode()))
+		req, err := http.NewRequest("POST", "http://statistics.data.gov.uk/sparql", strings.NewReader(v.Encode()))
+		if err != nil {
+			log.Event(ctx, "error returned from building a request for the children of an area", log.ERROR, log.Error(err))
+			return nil, err
+		}
+
 		req.Header.Add("Accept", "application/sparql-results+json")
-		resp, _ := client.Do(req)
+		resp, err := client.Do(req)
+		if err != nil {
+			log.Event(ctx, "error returned from the POST request", log.ERROR, log.Error(err))
+			return nil, err
+		}
```

Comment:
Can you:
- Move `defer resp.Body.Close()` to before `ioutil.ReadAll(resp.Body)` 
- Check the response status before you unmarshall the response.

Reply:
Thank you, I have updated this

Reply:
Same comment applies to this defer close too.